src/pair_hybrid_scaled.cpp

```diff
@@ -32,9 +32,14 @@ using namespace LAMMPS_NS;
 /* ---------------------------------------------------------------------- */
 
 PairHybridScaled::PairHybridScaled(LAMMPS *lmp) :
-    PairHybrid(lmp), fsum(nullptr), tsum(nullptr), scaleval(nullptr), scaleidx(nullptr)
+    PairHybrid(lmp), fsum(nullptr), tsum(nullptr), scaleval(nullptr), scaleidx(nullptr),
+    atomvar(nullptr), atomscale(nullptr)
 {
   nmaxfsum = -1;
+
+  // set comm size needed by this Pair (if atomscaleflag)
+
+  comm_forward = 1;
```

Comment:
Shouldn't `comm_forward = 1` depend on `atomscaleflag`? Or it doesn't matter?

Reply:
Yes, it could be set to zero if not `atomscaleflag`, but we don't set `atomscaleflag` until inside `compute()`. If I understand correctly, the size of `comm->maxforward` will usually be greater than 1, so there is no extra memory allocated.